data/Lookup.txt

```diff
@@ -0,0 +1,112 @@
+Week 1 Mon	:	12/8/2019
+Week 1 Tue	:	13/8/2019
+Week 1 Wed	:	14/8/2019
```

Comment:
@Krazzen @darrenoje @lionlim97 @0325961 @hwbjerry 
Will this data be a part of the JAR file you create? If not, when you run the program on a different machine, you may end up with errors. Please take note.